tsp_cli_client

```diff
@@ -370,20 +372,35 @@ if __name__ == "__main__":
             __get_tree(options.uuid, options.get_xy_tree, "TREE_TIME_XY")
 
         if options.get_xy:
-            if not options.items or not options.times:
-                print("Provide requested --items and --times for the XY data")
+            if not options.items:
+                print("Provide requested --items for the XY data")
+                sys.exit(1)
+
+            if not options.times and not options.time_range:
+                print("Provide either requested --times or --time-range for the XY data")
                 sys.exit(1)
 
             if options.uuid is not None:
-                parameters = {TspClient.REQUESTED_TIME_KEY: list(map(int, options.times)),
-                              TspClient.REQUESTED_ITEM_KEY: list(map(int, options.items))}
+                parameters = {TspClient.REQUESTED_ITEM_KEY: list(map(int, options.items))}
+                if options.times:
```

Comment:
this option is not supported by the TSP. According to TSP only time-range is supported. I think it should be removed.